daemon/execdriver/native/create.go

```diff
@@ -222,3 +227,17 @@ func (d *driver) setupLabels(container *libcontainer.Config, c *execdriver.Comma
 
 	return nil
 }
+
+func (d *driver) setupAdditionalGroups(container *libcontainer.Config, c *execdriver.Command) error {
+	container.AdditionalGroups = make([]int, len(c.GroupAdd))
+
+	for i, g := range c.GroupAdd {
+		group, err := strconv.Atoi(g)
```

Comment:
@thockin I expect review comments that we should probably accept group names. If not, then I can modify the code to use []int all the way up the stack.